02.calendar/cal.rb

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env ruby
+require 'date'
+require 'optparse'
+
+params = ARGV.getopts("y:m:")
+if !params["y"].nil? && !(1..9999).include?(params["y"].to_i)
+  puts "年は1~9999の中で指定してください" 
+  exit
+end
+if !params["m"].nil? && !(1..12).include?(params["m"].to_i)
+  puts "月は1~12の中で指定してください"
+  exit
+end
+
+if params["y"].nil?
+  year = Date.today.year
+else
+  year = params["y"].to_i
+end
```

Comment:
```suggestion
year = params["y"] ? params["y"].to_i : Date.today.year
```

こんなふうに三項演算子を使うと短く書けます

Reply:
ありがとうございます！三項演算子を使う形で修正しました。